### DISCUSSION TITLE
Correct way to configure a preset? (Using noice with LazyVim)

### DISCUSSION THREAD

**Discussion by ddbrierton** [folke/noice.nvim#459] [Q&A] [answered]
I’m very new to nvim, so apologies for what is a very basic question. I’m using LazyVim. I created a `plugins/noice.lua` with this single preset setting, my understanding being that it would be merged into the default configuration:

```
return {
  "folke/noice.nvim",
  config = function()
    require("noice").setup({
      presets = {
        lsp_doc_border = true,
      },
    })
  end,
}
```

I’m clearly doing something wrong as whilst this does add a border to LSP documentation, the layout of the command box and its completions have now changed.

Scrrenshot before the above change:

<img width="1378" alt="before" src="https://github.com/folke/noice.nvim/assets/7255543/f945a43e-cdc2-4fcb-970e-23a8085944a7">

Screenshot after the above change:

<img width="1378" alt="after" src="https://github.com/folke/noice.nvim/assets/7255543/cace3573-8225-447a-829f-af8562fccd54">

**Comment by folke** [comment, marked as answer]
You're overriding the LazyVim defaults. If you want to keep them, do use `opts` instead of a `config` function

```lua
opts = {presets = {
        lsp_doc_border = true,
      },}
```

**Comment by folke** [comment]
Sorry, this is not the LAzyVim repo. Your top screenshot is also a preset, so you probably configured that already somewhere else and now overwrote that?

**Reply by ddbrierton** [reply]
Apologies, I wasn’t sure whether to ask here or on the LazyVim repo.
